frontend/tests/features/collection.test.ts

```diff
@@ -1,57 +1,77 @@
 import { ROUTES } from "src/common/constants/routes";
 import { apiTemplateToUrl } from "src/common/utils/apiTemplateToUrl";
 import { API_URL } from "src/configs/configs";
-import { TEST_ENV, TEST_URL } from "tests/common/constants";
-import { goToPage } from "tests/utils/helpers";
-import { getText } from "tests/utils/selectors";
+import { TEST_URL } from "tests/common/constants";
+import { goToPage, login } from "tests/utils/helpers";
+import { getTestTag, getText } from "tests/utils/selectors";
 
-const describeIfCalledByDevEnv = TEST_ENV === "dev" ? describe : describe.skip;
+// (thuang): TEMP
+// Use `TEST_ENV.includes("local")` later
+const describeIfDeployed = describe.skip;
```

Comment:
Skipping logged in tests for now, and @seve will help add this back when he ports over AuthN flow from Explorer and make sure the login flow works for both Auth0 and mocked Happy Path AuthN page 🙆‍♂️